Python/specialize.c

```diff
@@ -357,14 +365,81 @@ _Py_Specialize_LoadAttr(PyObject *owner, _Py_CODEUNIT *instr, PyObject *name, Sp
     }
 
 fail:
-    STAT_INC(specialization_failure);
+    STAT_INC(loadattr_specialization_failure);
     assert(!PyErr_Occurred());
     cache_backoff(cache0);
     return 0;
 success:
-    STAT_INC(specialization_success);
+    STAT_INC(loadattr_specialization_success);
     assert(!PyErr_Occurred());
     cache0->counter = saturating_start();
     return 0;
 }
 
+
+int
+_Py_Specialize_LoadGlobal(
```

Comment:
Are there any cases where we return something other than `0`?  It seems like there are not.  So why are we returning int?

Reply:
Consistency mainly. `_Py_Specialize_LoadAttr()` can return -1, so it is easier to maintain if all `_Py_Specialize_XXX` functions all return the same error codes.